util.go

```diff
@@ -318,6 +318,32 @@ func (port *PortInfo) checkLinkStatus() error {
 	return nil
 }
 
+// handleLacpUpdate
+func (link *LinkInfo) handleLacpUpdate(lacpActive bool) {
+	log.Infof("Handling LACP update for link: %s, Lacp status: %+v", link.Name, lacpActive)
+	port := link.Port
+	if !lacpActive {
+		for idx, activeLink := range port.ActiveLinks {
+			if link == activeLink {
+				if idx == len(port.ActiveLinks) {
```

Comment:
isn't this len(xxx) -1 ?
if you don't want to keep the order then
ActiveLinks[i] = ActiveLinks[len(ActiveLinks)-1] 
ActiveLinks = ActiveLinks[:len(ActiveLinks)-1]